src/sentry/options/defaults.py

```diff
@@ -413,7 +413,7 @@
 
 # Post process forwarder options
 # Gets data from Kafka headers
-register("post-process-forwarder:kafka-headers", default=False)
+register("post-process-forwarder:kafka-headers", default=True)
```

Comment:
Changing the default value here would make all environments start using the headers including ST and Self hosted. Those environments don't need to use the header. Can we change the option only in the SaaS environment?

Reply:
Why not using the header in other environments? Some single tenant customers have been having a lot of problems with the post process forwarder and we wanted to give this a go since it helped in SaaS. This was actually the motivation to change the default.

Reply:
Discussed offline and this is ok